app/src/ui/multi-commit-operation/cherry-pick.tsx

```diff
@@ -130,6 +130,7 @@ export abstract class CherryPick extends BaseMultiCommitOperation {
         defaultBranch={defaultBranch}
         upstreamDefaultBranch={upstreamDefaultBranch}
         upstreamGitHubRepository={upstreamGhRepo}
+        gitHubAccount={null}
```

Comment:
How can I get the user's GitHub account info in this file? It's necessary for querying the API in the create branch component. I wasn't able to figure out how this file is even called.

Reply:
You'd start by passing it from the App state to the `MultiCommitOperation` component here:

https://github.com/desktop/desktop/blob/291d632825a349856bc1f49b77104705e745473d/app/src/ui/app.tsx#L2187

And then to the `CherryPick` component here

https://github.com/desktop/desktop/blob/291d632825a349856bc1f49b77104705e745473d/app/src/ui/multi-commit-operation/multi-commit-operation.tsx#L17

Reply:
Perfect, added in https://github.com/desktop/desktop/pull/16904/commits/d9e3c23a2bcefc6d1abcc96cbc3420b80e5325b2, thanks!